### PR TITLE
Debug

### PR DESCRIPTION
**Pull request by paulsonnentag**
add a raw view that renders the underlying json of a document

Screen Shot 2022-11-25 at 09 47 27

### CODE REVIEW

File: src/shared-worker.ts [hunk 1]
```diff
@@ -38,8 +38,8 @@ self.Automerge = Automerge
 // $ mkdir .amrg
 // $ yarn start
 // uncomment this and comment out the one below:
-// const url = "ws://localhost:3030" // local sync server
-const url = "ws://67.207.88.83" // cloud sync server on DigitalOcean
+const url = "ws://localhost:3030" // local sync server
```
Comment (pvh) [inline src/shared-worker.ts]: oops?

File: src/components/content-types/workspace/Workspace.tsx [hunk 2]
```diff
@@ -81,6 +81,11 @@ export default function Workspace({
       return
     }
 
+    // don't add raw view urls
+    if (parseDocumentLink(currentDocUrl).type === "raw") {
```
Comment (pvh) [inline src/components/content-types/workspace/Workspace.tsx]: this violates the "never check a type name" principal

**Comment by pvh** [comment]
This is something I'd like to merge but I don't think we should land the patch without a bit more work on it.

File: src/components/content-types/workspace/Workspace.tsx (hunk 2 above)
Comment (paulsonnentag) [reply src/components/content-types/workspace/Workspace.tsx]: I've turned it into a config option in content type

**Pull request merged**